### DISCUSSION TITLE
folium.LayerControl for two layers with points

### DISCUSSION THREAD

**Discussion by jllucero** [opengeos/leafmap#62] [Q&A] [answered]
Hello.  I'm encountering a new issue.  I have several boundary layers in my map and one layer consists of points.  They all plot fine and the layer control icon in the upper right corner works as it should.  However, when I add a new layer of points I not only lose the layer control icon but I also can't map two layers of points.  For the layer with points, I define a feature group, add this as a layer, add my points to it and then add the layer to the map with this statement "folium.LayerControl().add_to(Map)"  This process works for one layer of points but not for  two.

I'd appreciate some suggestions.  

Thank you.

**Comment by giswqs** [comment, marked as answer]
Folium has very limited interactive functionality. You need to add all the data layers before displaying the map. Once the map is displayed, you CANNOT add additional layers to it. Try the ipyleaflet plotting backend. That gives you a lot more flexibility for modifying the map. Try it out with Binder: https://gishub.org/leafmap-pangeo

**Reply by jllucero** [reply]
Thank you!  Actually, "Map.add_gdf" works perfectly to add multiple layers of points.

However, now that I'm using "Map.add_gdf" to plot my points in a layer, can I change the marker size as a parameter?  Previously, I was able to change the style but it was by using a feature group which got me in this issue to begin with.  Here's how my map looks.  I'd appreciate any suggestion on how to correct this.
Thank you.
![Icons](https://user-images.githubusercontent.com/56424972/123696571-b6083680-d810-11eb-9ca7-139e9886ab39.PNG)
